src/index.js

```diff
@@ -109,7 +109,7 @@ todoListItem.addEventListener('keydown', (event) => {
   }
 });
 clearButton.addEventListener('click', () => {
-  checkboxCheck();
+  checkboxCheck(todoItems);
```

Comment:
- [ ] I would recommend you use a change event listen for the checkbox as it is what is been used for inputs and it is because it actually changes according to the user's choice.